src/json.rs

```diff
@@ -27,15 +27,9 @@ mod export {
     pub type JsonMap = simd_json::owned::Object;
     pub const NULL: Value = Value::Static(simd_json::StaticNode::Null);
 
-    pub use simd_json::{
-        json,
-        Builder,
-        Error as JsonError,
-        Mutable,
-        StaticNode,
-        Value as ValueTrait,
-        ValueAccess,
-    };
+    pub use simd_json::base::{ValueAsContainer, ValueAsMutContainer, ValueAsScalar};
+    pub use simd_json::derived::TypedContainerValue;
```

Comment:
```suggestion
    pub use simd_json::prelude::{
        TypedContainerValue,
        ValueAsContainer,
        ValueAsMutContainer,
        ValueAsScalar,
    };
```
meh, a little cleaner this way. Any future traits needed can also be imported from the prelude.